src/mono/sample/wasm/browser-threads/main.js

```diff
@@ -61,10 +61,12 @@ function onInputValueChanged(exports, inputElement) {
 
 try {
     const inputElement = document.getElementById("inputN");
-    const { runtimeBuildInfo, setModuleImports, getAssemblyExports, runMain } = await createDotnetRuntime(() => {
-        console.log('user code in createDotnetRuntime callback');
-        return {
-            configSrc: "./mono-config.json",
+    const { setModuleImports, getAssemblyExports, runMain } = await dotnet
+        .withModuleConfig({
```

Comment:
shall we simplify the basic sample and move this to `browser-advanced` sample ?

Reply:
specificaly threads sample doesn't need it, I think

Reply:
We have templates that shows the simplest approach. I don't mind have this example section in samples.